src/components/tree/TreeInnerNode.svelte

```diff
@@ -28,7 +30,7 @@
   {#if expanded}
     {#each node.datasets as child (child.title)}
       {#if child instanceof DataSet}
-        <TreeLeafNode node={child} />
+        <TreeLeafNode {chart} node={child} />
```

Comment:
The Svelte linter actually reverts this change when I try to apply it :) I could disable this in the config, but shorthand attributes like this show up a lot in the code base, e.g. a couple lines before:
```
<side class="left" {style} data-tour="browser">
                   ^^^^^^^
```
and elsewhere:
```
<select class="uk-select" required {name} {id} bind:value>
                                   ^^^^^^ ^^^^
```
So I think it might be fairly clear what's going on with these.

Reply:
i dont love it, but if thats the preferred style in Svelte, we might as well go with it.